### PR TITLE
Fix nightly content instalation

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [x] Ready

## Description
Changed content instalation to run parallel on all servers when running nightly.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -403,7 +403,6 @@ jobs:
                 echo "Skipping - not running in INSTANCE_TESTS build"
                 exit 0
             fi
-            ./Tests/scripts/slack_notifier.sh $CIRCLE_NODE_INDEX ./env_results.json
```
Comment (yaakovi) [inline .circleci/config.yml]: why did you remove this?

Comment (ShahafBenYakir) [reply .circleci/config.yml]: I didn't want to spam the content team channel so I removed the notifications while I fixed this. Good thing you caught it.
